scripts/qa_crowbarsetup.sh

```diff
@@ -2436,7 +2436,7 @@ function enable_ssl_generic()
             return
         ;;
         heat)
-            if [[ ! $(iscloudver 7plus) ]] ; then
+            if ! iscloudver 7plus ; then
```

Comment:
oh, actually @bmwiedemann is right, the return can't be at the end of the `if`. Is correct how it is now, or we can add the `return` in a `else`